cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -83,7 +100,29 @@ def _run_task(self):
             os.path.join(self.working_path, options.get("outputdir", ".")), os.getcwd()
         )
 
-        num_failed = robot_run(self.options["suites"], **options)
+        if self.options["processes"] > 1:
+            cmd = [
+                sys.executable,
+                "-m",
+                "pabot.pabot",
+                "--pabotlib",
+                "--processes",
+                str(self.options["processes"]),
+            ]
+            # We need to convert options to their commandline equivalent
+            for option, value in options.items():
+                if isinstance(value, list):
+                    for item in value:
+                        cmd.extend([f"--{option}", str(item)])
```

Comment:
```suggestion
                        cmd.extend(["--{option}", str(item)])
```

Only use f-strings when you're actually substituting something; otherwise flake8 complains (are you not running flake8?)

Reply:
This is substituting something:  `{option}`. Maybe I should rename that to `{option_name}` to make it more obvious.

Reply:
@boakley sorry, my mistake, not sure what I was thinking